python/arcticdb/arctic.py

```diff
@@ -149,8 +148,13 @@ def __init__(self, uri: str, encoding_version: EncodingVersion = EncodingVersion
         self._library_adapter = _cls(uri, self._encoding_version)
         self._library_manager = LibraryManager(self._library_adapter.config_library)
         self._uri = uri
+        self._open_libraries = dict()
 
     def __getitem__(self, name: str) -> Library:
+        already_open = self._open_libraries.get(name)
```

Comment:
`get(name, None)`

Reply:
That's the same as I've written https://docs.python.org/3/library/stdtypes.html?highlight=dict%20get#dict.get

Reply:
Ah, I've always thought the default is not optional...